### ISSUE TITLE
docs: Specify common rules

### ISSUE THREAD

**Issue by tbobm** [open-subjects/open-subjects#17]
# Common validation rules

Define expectations regarding:
- [x] Packaging
- [x] Linting
- [x] Documentation

TLDR: Validation requests must comply with 3 common principles.

## Packaging

- Program must create an executable (an entrypoint script may be used)
- Repository must include some sort of dependency lock file (soft dependencies shall be considered first)

## Linting

- Code must comply with at least one linting community guideline

## Documentation

- Repository must be properly documented
- Installation process must be properly documented for a least a single Linux-based OS (additional OSes may be documented)
- Execution process must be documented (either directly through a helper interface or in the repository documentation)

## Additional common goal

The implementation of a Continuous Integration mechanism is highly encouraged to enforce these expectations.
